src/Compilers/CSharp/Portable/Lowering/SynthesizedMethodBaseSymbol.cs

```diff
@@ -156,6 +157,14 @@ public sealed override ImmutableArray<CSharpAttributeData> GetReturnTypeAttribut
             return InheritsBaseMethodAttributes ? BaseMethod.GetReturnTypeAttributes() : ImmutableArray<CSharpAttributeData>.Empty;
         }
 
+#nullable enable
+        public sealed override DllImportData? GetDllImportData() => InheritsBaseMethodAttributes ? BaseMethod.GetDllImportData() : null;
+
+        internal sealed override MethodImplAttributes ImplementationAttributes => InheritsBaseMethodAttributes ? BaseMethod.ImplementationAttributes : default;
+
+        internal sealed override MarshalPseudoCustomAttributeData? ReturnValueMarshallingInformation => InheritsBaseMethodAttributes ? BaseMethod.ReturnValueMarshallingInformation : null;
```

Comment:
>internal sealed override MarshalPseudoCustomAttributeData? ReturnValueMarshallingInformation => InheritsBaseMethodAttributes ? BaseMethod.ReturnValueMarshallingInformation : null; [](start = 8, length = 179)

Should we override HasDeclarativeSecurity?

Reply:
And other APIs related to security attributes.

---
In reply to: [354052802](https://github.com/dotnet/roslyn/pull/40011#discussion_r354052802) [](ancestors = 354052802)

Reply:
I think these attributes are not supported in .NET Core, so I wasn't sure if it made sense to spend time to make them work.

---
In reply to: [354053505](https://github.com/dotnet/roslyn/pull/40011#discussion_r354053505) [](ancestors = 354053505,354052802)

Reply:
> I think these attributes are not supported in .NET Core, so I wasn't sure if it made sense to spend time to make them work.

Well-known attributes are not blessed in any way and don't have to be defined in the framework. Anyone can define them. If we decide to not support them on local functions, then we should report errors if they are applied.

---
In reply to: [354066372](https://github.com/dotnet/roslyn/pull/40011#discussion_r354066372) [](ancestors = 354066372,354053505,354052802)

Reply:
Adding a PROTOTYPE comment

---
In reply to: [354068222](https://github.com/dotnet/roslyn/pull/40011#discussion_r354068222) [](ancestors = 354068222,354066372,354053505,354052802)